dudendown.py

```diff
@@ -278,9 +278,12 @@ def main():
 
     # 2. Parse synonyms section
     # currently unused
-    syn_section = smpage[Sections.synonyms.value]
+    try:
+        syn_section = smpage[Sections.synonyms.value]
 
-    syn_array = recursively_extract(syn_section, extract_synonym_from_li)
+        syn_array = recursively_extract(syn_section, extract_synonym_from_li)
+    except:
```

Comment:
Could you specify which exceptions should be caught? Pep8 says that we [shouldn't use bare except](https://www.python.org/dev/peps/pep-0008/#programming-recommendations) (point 9):

> When catching exceptions, mention specific exceptions whenever possible instead of using a bare except: clause.